<!DOCTYPE HTML>
<html>
	<head>
		<title>Brandon M Lind</title>
		<meta http-equiv="content-type" content="text/html; charset=utf-8" />
		<meta name="description" content="" />
		<meta name="keywords" content="" />
		<meta name="google-site-verification" content="MB1Ogq2g9YG3iBO5-FvcB-Ng-oRRWmoQ9B1oPzHxJVI" />
		<link href='https://fonts.googleapis.com/css?family=Arimo:400,700' rel='stylesheet' type='text/css'>
		<!--[if lte IE 8]><script src="js/html5shiv.js"></script><![endif]-->
		<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js"></script>
		<script src="js/skel.min.js"></script>
		<script src="js/skel-panels.min.js"></script>
		<script src="js/init.js"></script>
		<script>(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');ga('create', 'UA-100959105-2', 'auto');ga('send', 'pageview');</script>
		<noscript>
			<link rel="stylesheet" href="css/skel-noscript.css" />
			<link rel="stylesheet" href="css/style.css" />
			<link rel="stylesheet" href="css/style-desktop.css" />
		</noscript>
		<!--[if lte IE 8]><link rel="stylesheet" href="css/ie/v8.css" /><![endif]-->
		<!--[if lte IE 9]><link rel="stylesheet" href="css/ie/v9.css" /><![endif]-->
	</head>
	<body class="homepage">

		<!-- Header -->
		<div id="header">
			<div class="container"> 
				
				<!-- Logo -->
				<div id="logo">
					<h1><a href="https://github.com/brandonlind/brandonlind.github.io/blob/master/research/images/banner.JPG?raw=true">Brandon M Lind</a></h1>
					<span>&nbsp;</span>
				</div>
				
				<!-- Nav -->
				<nav id="nav">
					<ul>
						<li><a href="../index.html"><strong><font color="white">Homepage</strong></font></a></li>
						<li class="active"><a href="../research/research.html"><strong><font color="white">Research and Interests</strong></font></a></li>
						<li><a href="../cv/cv.html"><strong><font color="white">CV</strong></font></a></li>
						<!-- <li><a href="../hobbies/hobbies.html"><strong><font color="white">Hobbies</strong></font></a></li> -->
					</ul>
				</nav>
			</div>
		</div>

		<!-- Main -->
		<div id="main">
			<div class="container">
				<div class="row"> 

					<!-- Sidebar -->
					<div id="sidebar" class="4u">
						<section>
							<ul class="style">
								<li>
									<a href="images/lab_logo_transparent2.png"><img src="images/lab_logo_transparent2.png" width="375" height="400"></a>
									<h3><center>.</center></h3>
									<br />
									<a href="images/rcw.pdf"><img src="images/rcw.pdf", width="375" height="225"</a>
									<h3><center>The endangered longleaf pine and its threatened habitat specialist, the red-cockaded woodpecker</center></h3>
									<br />
									<a href="images/evome.pdf"><img src="images/evome.pdf", width="375", height="225"</a>
									<h3><center>Arctic grayling and the feltleaf willow</center></h3>
									<br />
									<a href="https://www.github.com/coadaptree/poolseq_pipeline"><img src="images/pipeline.png" width="375" height="225"></a>
									<h3><center>Public bioinformatics pipeline</center></h3>
									<br />
									<a href="images/all_species_plus_lp.pdf"><img src="images/all_species_plus_lp.pdf" width="375" height="200"></a>
									<h3><center>CoAdaptree Populations</center></h3>
									<p class="text">Dark Green: jack pine (<i>Pinus banksiana</i>)<br />
									Brown: lodgepole pine (<i>P. contorta</i>)<br />
									Purple: Interior Douglas-fir (<i>Pseudotsuga menziesii</i> var. <i>glauca</i>)<br />
									Light Green: Coastal Douglas-fir (<i>P. menziesii</i> var. <i>menziesii</i>)<br />
									Blue: Wester Larch (<i>Larix occidentalis</i>)<br />
									<br />
									<a href="images/allelic_clines.png"><img src="images/allelic_clines.png" width="375" height="200"></a>
									<h3><center>Genetic clines of adaptive loci</center></h3>
									<br />
									<a href="images/gypsy_map.pdf"><img src="images/gypsy_map.pdf" width="375" height="315"></a>
									<h3><center>Gypsy moth invasion across Eastern USA</center></h3>
									<br />
									<a href="https://github.com/brandonlind/brandonlind.github.io/blob/master/research/images/3d_plot_map.png?raw=true" class="image full"><img src="images/3d_plot_map.png?raw=true" alt="" /></a>
									<p class="text">Blue: red fir (<i>Abies magnifica</i>)<br />
									Gray: white fir (<i>A. concolor</i>)<br />
									Red: Jeffrey pine (<i>Pinus jeffreyi</i>)<br />
									Green: sugar pine (<i>P. lambertiana</i>)<br />
									Orange: incense cedar (<i>Calocedrus decurrens</i>)<br />
									Black: Oak/Willow/other</p>
									<h3><center>3D stem map at Teakettle Forest</center></h3>
									<br />
									<a href="images/lake_tahoe.pdf" ><img src="images/lake_tahoe.pdf" width="350" height="425" alt="" /></a>
									<h3><center>Sierra Nevada Rain shadow at Lake Tahoe</center></h3>
									<br />
									<a href="images/consensus_map.png" ><img src="images/consensus_map.png" width="350" height="350" alt="" /></a>
									<h3><center>Foxtail pine linkage map</center></h3>
									<br />
									<a href="images/polygenic_evolution.png" ><img src="images/polygenic_evolution.png" width="350" height="425" alt=""/></a>
									<h3><center>Evolution of polygenic traits</center></h3>
								</li>
							</ul>
						</section>
					</div>
					
					<!-- Content -->
					<div id="content" class="8u skel-cell-important">
						<section>
							<header>
								<!-- <h3><font size="6">General Research Interests</h3></font> -->
								<h2>General Research Interests</h2>
								<p align="justify">My research program aims to understand how species adapt to environmental stressors such as climate change, pathogens, and invasive species, and how they respond to ongoing pressures such as habitat loss. I integrate field experiments, population and comparative genomics, and computational modeling to uncover the molecular mechanisms driving adaptation and resilience, as well as the evolutionary processes that drive population and species differences. Focal species of my research program range from forest trees and their associated pests and pathogens, to Arctic willow and salmonid species, to several rare species of high conservation priority such as the red-cockaded woodpecker, longleaf pine, and whitebark pine. A central goal of my research is to translate genomic insights into practical tools for biodiversity conservation and ecosystem management. By bridging molecular biology, ecology, and data science, my work addresses both basic and applied outcomes for sustaining biodiversity in a changing world.</p>
							</header>
							<header>
								<!-- <h3><font size="6">Dissertation background</h3></font> -->
								<h2>Research Projects</h2>
							</header>
							<header>


							<h3>Training future scientists through conservation genomics: longleaf pine and red-cockaded woodpecker genome projects</h3>
							In collaboration with the University of Florida, the US Fish & Wildlife Service, and Archbold Biological Station, I am directing the assembly of the first reference genomes for two ecologically critical and imperiled species: the endangered longleaf pine (<i>Pinus palustris</i>) and the threatened red-cockaded woodpecker (<i>Leuconotopicus borealis</i>). These species co-occur in fire-adapted Southeastern U.S. ecosystems, where the red-cockaded woodpecker is dependent upon mature longleaf pine stands. Supported by funding awarded through ORG.one, this project has been integrated into the Biology and Conservation Genomics undergraduate training program at UConn - a year-long program that trains several undergraduate students in bioinformatics, genome assembly, and conservation biology. By actively engaging with conservation practitioners, students gain hands-on learning while contributing critical genomic resources to support species recovery and ecosystem management. At the end of the program the students will publish a peer-reviewed article describing the assembly and insights gained through comparative genomics.
							<br /><br />



							<h3>The Evolving Meta-ecosystems Biology Integration Institute</h3>
							As part of the NSF-funded Evolving Meta-Ecosystems Biology Integration Institute, my research explores how rapid climate change shapes evolutionary processes and species interactions across Arctic river and tundra ecosystems—some of the most rapidly warming systems on Earth. I focus on two keystone species: Arctic grayling (<i>Thymallus arcticus</i>) and feltleaf willow (<i>Salix alaxensis</i>), both of which are locally adapted and central to Arctic community dynamics. Using long-read sequencing, I am assembling the first reference genomes for these species and several related willows to investigate conserved regions, gene family evolution, and structural variation. To better capture species-level diversity, I am constructing reference-aided pangenome graphs that integrate landscape genomic sampling and common garden phenotypes. Ultimately, this work contributes to trait-based predictions within ecological models, offering insight into how evolutionary processes shape the structure and function of complex ecosystems.
							</i>
							<br /><br />




							<h3>The accuracy of predicting maladaptation to new environments with genomic data</h3>
							Methods using genomic information to forecast population maladaptation to climate change (i.e., genetic offset methods) are becoming increasingly common, yet there has been little exploration into the factors that could potentially affect performance of these methods, or mislead inference with regard to model accuracy during validation. These knowledge gaps pose serious hurdles toward the incorporation of such methods into management and policy. In Dr. Lotterhos' lab at Northeastern, I used extensive sets of simulations to understand the circumstances in which these methods will be most accurate. We found several factors related to the evolutionary history of targeted populations, as well as the experimental design used to train models, that affect model accuracy. Our work was featured in <a href="../cv/pdfs/Lind_and_Lotterhos_2024_The_accuracy_of_predicting_maladaptation_to_new_environments_Mol_Ecol_Res.pdf">Molecular Ecology Resources</a> on the <a href="https://onlinelibrary.wiley.com/doi/epdf/10.1111/1755-0998.13973">issue's cover</a>, and is part of a special issue in the <a href="../cv/pdfs/Lind_and_Lotterhos_2025_A_comparison_of_genomic_forecasts_based_on_genotypes_versus_allele_frequencies.pdf">American Naturalist</a>.
							<br /><br />

							<h3>The CoAdapTree Project</h3>
							My first postdoc was at the University of British Columbia working on the <a href="http://coadaptree.forestry.ubc.ca">CoAdapTree project</a> with Sally Aitken and Sam Yeaman. This project examines how biotic (eg pathogens) and abiotic (eg climate) factors influence genomic variation across four conifer species. My primary responsibilities focussed on <a href="../cv/pdfs/Lind_et_al_2021-Haploid-diploid-and-pooled-exome-capture.pdf">developing genetic resources</a> to describe the adaptive genetics of fungal- (e.g., <a href="../cv/pdfs/New Phytologist - 2024 - Singh - Genetic architecture of disease resistance and tolerance in Douglas-fir trees.pdf">swiss needle cast and rhabdocline</a> and <a href="../cv/pdfs/Lu_et_al_2025_Genetic_Architecture_Underlying_Response_to_the_Fungal_Pathogen_Dothistroma.pdf">dothistroma needle blight</a>), cold-, and drought-hardiness in 70 Douglas-fir and 87 western Larch populations, as well as climate adaptation in these two species, lodgepole pine, and jack pine (see map to left). To handle all of this data, and to do so with reproducibility in mind, I've made my bioinformatics pipeline <a href="https://www.github.com/coadaptree/poolseq_pipeline">publically available</a>. In fact, I've made all of the code from each lead-authored project available on GitHub, including our <a href="https://github.com/brandonlind/cmh_test">case-control</a> genotype-phenotype method. See Data Availability statements within each manuscript.
							</p>

							<h3>The genetic architecture of repeated local adaptation to climate in distantly related plants <a href="../cv/pdfs/Whiting_et_al_2024_the_genetic_architecture_of_repeated_local_adaptation_Nature_eco_evo.pdf"><font color="blue">PDF</font></a></h3>
							How repeatable is evolution? This question, popularized by Stephen Jay Gould’s ‘replaying the tape of life’ thought experiment, has been the subject of decades of empirical research. Reanalyzing genomic data from thousands of individuals across 25 plant species, I contributed to a study that identified 108 gene families repeatedly linked to climate adaptation. These genes will be crucial for monitoring in breeding and natural populations to assess the how well populations are maintaining climate adaptation. This study also highlights the advantage of phylogenetic analyses to understand adaptation.</p>

							<h3>How useful is genomic data for predicting maladaptation to future climate? <a href="../cv/pdfs/Lind-et-al-2024_How-useful-is-genomic-data-for-predicting-maladaptation-to-future-climate_Global-Change-Biology.pdf"><font color="blue">PDF</font></a></h3>
							<p align="justify">The pace of climate change threatens to decouple adaptive variation and environmental optima of many natural populations. Characterising the vulnerability of populations across a species range can be particularly useful to direct management priorities. Over the past decade, using genomic data to predict maladaptation of natural populations to future climate (i.e., genomic offset methods) have become increasingly widespread but with few instances where model predictions are validated with ground truth data. We use range-wide data across three conifer taxa to assess such methods. We validate these methods against two-year and 52-year growth and mortality measured in independent transplant experiments. Overall, we find that two common methods often better predict transplant performance than climatic or geographic distances. We also find that these models are surprisingly not improved using GEA candidates. Even with promising validation results, variation in model projections to future climates can make it difficult to identify the most maladapted populations using either method.</p>
							
							<h3>Evaluating genomic data for management of local adaptation in a changing climate <a href="../cv/pdfs/Mahoney_et_al_2019_Evaluating_genomic_data_for_management_of_local_adaptation_in_a_changing_climate_A_lodgepole_pine_case_study.pdf"><font color="blue">PDF</font></a></h3>
							<p align="justify">Understanding how managed populations are adapted to their local environments will assist in both breeding and reforestation efforts by identifying environmental sources driving trait differentiation and pathogen susceptibility. To feasibly (weighing time, effort, and cost) disect these aspects of plant biology, there are a few common approaches that can be considered, but do they hold the same information content with regard to the environmental drivers of differentiation or the spatial scale of adaptation? To address this, we compared climate data, genomic data, and a long term provinance trial in lodgepole pine in this regard. We discuss relative strengths and weaknesses when considering the appropriate data for managing populations with broad geographic ranges. (see figure to the left showing range-wide clines in genomic regions underlying adaptation)</a></p>

							<h3>Evolutionary genomics of the invasive and defoliating gypsy moth <a href="../cv/pdfs/Friedline_et_al-2019-Evolutionary_genomics_of_gypsy_moth_populations_sampled_along_a_latitudinal_gradient-Molecular_Ecology.pdf"><font color="blue">PDF</font></a></h3>
							<p align="justify">Biological invasions from non-native species can have have large economic consequences and can fundamentally alter the ecology of the invaded system. As one such example, the European gypsy moth was introduced to the eastern United States around the late 19th century and has since spread north through Maine, west through Wisconsin, and south through the Carolinas. Consequently, the gypsy moth has become one of the most destructive hardwood pests in North America. To understand how this species has been so successful in a novel environment we sampled populations from the range edges by examining genomic variation underlying trait differentiation for three developmental traits. We described how the invasion was facilitated through adaptive variation found throughout the genome and that genomic patterns (ie linkage disequilibrium) among adaptive genetic componants were driving differences in the three traits among populations. (see map to the left showing the chronological spread of the gypsy moth across New England) </p><br />
							<h3>Effects of Forest Management on Fine-scale Gene Flow in a Fire-suppressed population of <i>Pinus lambertiana</i> Dougl. <a href="../cv/pdfs/Lind_et_al_2019_Effect_of_fire_and_thinning_on_fine-scale_genetic_structre_and_gene_flow_in_fire-suppressed_populations_of_sugar_pine_Forest_Ecology_And_Management.pdf"><font color="blue">PDF</font></a></h3>
							<a href="images/yosemite_valley.png" class="image full"><img src="images/yosemite_valley.png" alt="" /><p>Effect of fire supression on forest crowding in the upper Yosemite Valley from Columbia Point (source unknown)</p></a>
							<p align="justify">Climate change is exacerbating the effects of fire suppression in the west, and prescribed fire and forest thinning are two ways of mitigating these effects. Sugar pine is an important species in these forests, and was historically dominant before fire suppression. We looked at how these treatments affect mating patterns in this species and discuss how this may affect long-term forest dynamics. (see 3D stem map to the left; more about Teakettle Experimental Forest <a href="../teakettle/teakettle.html"><font color="blue">here</font></a>).</p>
							<br />
							<h3>Local adaptation of the endangered whitebark pine across fine spatial scales of the Lake Tahoe Basin - <a href="../cv/pdfs/Lind_et_al_2017_Water-availability-drives-signatures-of-local-adaptation-in-whitebark-pine-across-fine-spatial-scales-of-the-lake-tahoe-basin_Molecular_ecology.pdf"><font color="blue">PDF</font></a></h3><br>
							Whitebark pine (<i>Pinus albicaulis</i>) is a foundational species in many high elevation forests and plays an important role in ecosystem function and services. Despite its influential role in these systems, whitebark pine is listed as a threatened species by IUCN. Therefore, understanding the spatial scale of adaptation in this species is vital for conservation efforts. Even so, much of our knowledge regarding adaptation in trees comes from broad geographic ranges, often beyond the scale at which most forest management is applied. Using a common garden we found that whitebark pine is adapted across the fine spatial scales of the Lake Tahoe Basin despite highly shared genetic variation across populations. We further showed that the adaptive signal driven by water availability is apparent across common garden and genomicw data, likely due to the rain shadow on the eastern Siera Nevada. (to the left see map of sampled populations from the Lake Tahoe Basin, and how quickly the rainshadow reduces annual preciptation by 50% in just under 15 miles)<br />
							<br />
							<br />
							<h3>Linkage maps and the genomic architecture of local adaptation</h3><br>
							For non-model organisms, genomic resources needed to fully describe the genetic and genomic architectures of complex traits can face many hurdles. Because of their large genome size, conifers face many challenges in this regard, particularly with respect to the production of physical sequence maps with ordered genetic markers. Such resources can be used to test fundamental theory regarding the expected genetic architecture underlying adaptive traits under various demographic and evolutionary scenarios. To produce such resources, linkage maps are often used to both order and elongate blocks of sequence data. Across taxa, such maps were predominately made from well understood familial relationships or mating designs, but maps of single trees began increasing in numbers in the 1990s. In conifers, maternally derived haploid tissue (i.e., a single copy of the genome from megagametophye tissue) can be used to order markers within individual trees within a fairly straightforward framework. To begin to test fundamental theories of local adaptation in a non-model conifer, <i>Pinus balfouriana</i>, we constructed a <a href="https://treegenesdb.org/cmap?id=15">dense linkage map</a> using individuals collected from populations that historically exhibited gene flow but have subsequently become isolated from one another since the last glacial maximum. <a href="../cv/pdfs/Friedline_et_al_2015_The-genetic-architecture-of-local-adaptation-I--the-genomic-landscape-of-foxtail-pine-as-revealed-from-a-high-density-linkage-map_Tree_genetics_and_genomes.pdf"><strong><font color="blue">PDF</font></strong></a><br />
							<br />
							After completing this linkage map, we used niche modeling and indirect measures of WUE, the ratio of carbon isotopes fixed during photosynthesis (&delta;<sup>13</sup>C), as well as nutrient utilization, the ratio of nitrogen isotopes fixed during leaf-level resource utilization (μgN), to explore the importance of water availability as a determinant of geographical range of foxtail pine. Additionally, we explored the degree to which variation in &delta;<sup>13</sup>C and μgN is genetically based, differentiated among populations, and underlain by a genetic architecture with large effect loci. <a href="https://www.biorxiv.org/content/biorxiv/early/2016/01/29/038240.full.pdf"><strong><font color="blue">PDF</font></strong></a><br />
							<br />
							<br />
							<h3>Current state and future directions of forest tree genomics <a href="../cv/pdfs/Lind_et_al_2018_The-genomics-of-local-adaptation--are-we-out-of-the-woods-yet_Tree_Genetics_and_Genomes.pdf"><font color="blue">PDF</font></a></h3><br />
</p>
						</section>
					</div>
					
				</div>
			</div>
		</div>

		<!-- Footer -->
		<div id="featured">
			<div class="container">
				<div class="row">
					<div class="4u">
						<h2><p align="center">Follow me on Bluesky</p></h2>
						<a href="https://brandonlind.bsky.social" class="image centered"><img src="../images/bluesky-24460-big.png" alt="" width="165" height="165"/></a>
					</div>
					<div class="4u">
						<h2><p align="center">Follow me on GitHub</p></h2>
						<a href="https://github.com/brandonlind" class="image centered"><img src="../images/github.png?raw=true" alt="" width="200" height="200"/></a>
					</div>
					<div class="4u">
						<h2><p align="center">Google Scholar</p></h2>
						<span><font size="1">&nbsp;</font></span>
						<a href="https://scholar.google.com/citations?user=O1EU6pYAAAAJ&hl=en&oi=ao" class="image centered"><img src="../images/gscholar.png?raw=true" alt="" width="160" height="160"/></a>
					</div>
				</div>
			</div>
		</div>


		<!-- Footer -->
		<div id="footer">
			<div class="container">
				<div class="row">
					<div class="4u">
						<section>
							<h2>Contact Info</h2>
							<ul class="default">
								<h3>Mail</h3>
								<li><p><font color="silver">Brandon Lind<br />
								Northeastern University<br />
								Marine and Environmental Science<br />
								Boston, MA 02115
								USA</font></p></li>
							</ul>
						</section>
					</div>
					<div class="4u">
						<section>
							<h2>&nbsp;</h2>
							<ul class="default">
								<h3>Email</h3>
								<li><a href="mailto:lind.brandon.m@gmail.com"><font color="silver">lind dot brandon dot m at gmail dot com</font></a></li>
							</ul>
						</section>
					</div>
				</div>
			</div>
		</div>

		<!-- Copyright -->
		<div id="copyright">
			<div class="container">
				Design: <a href="https://creativecommons.org">Creative Commons</a> Images: <a href="index.html">© Brandon Lind 2017</a>
			</div>
		</div>
		
	</body>
</html>